grid/__main__.py

```diff
@@ -44,6 +44,11 @@
     help="If this flag is used a SQLAlchemy DB URI is generated to use a local db.",
 )
 
+parser.add_argument(
+    "--id", type=str, help="PyGrid Node ID.", default=os.environ.get("NODE_ID", None),
+)
```

Comment:
Should this be formatted to match the rest of parser.add_argument like this?
```suggestion
parser.add_argument(
    "--id", 
    type=str, 
    help="PyGrid Node ID.", default=os.environ.get("NODE_ID", None),
)
```

Reply:
Apparently, black does not accept this formatting style. :/